sync_ci/pkg/parser/rules.go

```diff
@@ -76,3 +84,36 @@ func (t *envParser) parse(job string, lines []string) []string {
 	}
 	return res
 }
+
+func UpdateRules() error {
+	// assumed one level json dictionary
+	file, err := os.Open(EnvRuleFilePath)
+	if err != nil {
+		return err // file not exist
+	}
+	defer file.Close()
+
+	content, err := ioutil.ReadAll(file)
+	if err != nil {
+		return err // io error
+	}
+
+	newEnvRules := map[string]string{}
+	err = json.Unmarshal(content, &newEnvRules)
+	if err != nil {
+		return err // invalid json format
+	}
+	envRules = newEnvRules
```

Comment:
Rigorously it should add a read write lock, However you can ignore it in this case.